dev/README.md

```diff
@@ -1,13 +1,106 @@
 # DEV 
 
+## First time setup
+
+This section contains all the instructions to setup your environment in order to deploy the infrastructure locally. 
+
+First of all, you need to install minikube following [this](https://minikube.sigs.k8s.io/docs/start/) guide. (Tested with minikube v1.16.0 on Redhat 8.4)
+
+Once you have installed minikube, start a new cluster with 
+
+```bash 
+minikube --memory 8192 --cpus 4 start  --kubernetes-version=v1.20.0
+```
+
+You can change the memory and cpu settings according to your system. **Other versions of kubernetes have not been tested**.
+
+Enable the `ingress` and `ingress-dns` addons with the following commands: 
+
+```bash
+minikube addons enable ingress
+minikube addons enable ingress-dns
+```
+
+Create a namspace for `kafka`
+
+```bash
+kubectl create ns kafka
+```
+
+From the root of the project, deploy the kafka infrastructure with 
+
+```bash
+kubectl apply -f dev/kubernetes/ -n kafka
```

Comment:
Since all these resources are related to Kafka, what do you think about moving them to `dev/kubernetes/kafka` instead? This way, we can leverage the directory structure for future resources.